apps/condo/domains/notification/templates.js

```diff
@@ -67,16 +73,30 @@ async function renderTemplate (transport, message) {
             }
         } else if (message.lang === 'ru') {
             return {
-                subject: '🔑 Восстановление пароля',
```

Comment:
we don't need an emoji ? :(

Reply:
I put it here and then realized we needed more consistency - do we use them everywhere or not at all :)